op-acceptance-tests/tests/interop/proofs/funccall_dsl.go

```diff
@@ -0,0 +1,39 @@
+package proofs
+
+import (
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-service/apis"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lmittmann/w3"
```

Comment:
Just a note that there's a [new contract call dsl](https://github.com/ethereum-optimism/optimism/blob/develop/op-devstack/dsl/contract/call.go) being introduced into `op-stack` that we might want to migrate to.

Reply:
tbh, I'm not a fan of the new contract call dsl. I'll stick with the `funccall_dsl.go` unless you feel strongly about using the new one.